File: 110術科模擬試題/F.py
#110商業技藝競賽模擬試題
#Problem F 漢明距離（Hamming distance）--字串比對法
#Author: Yotrew Wing
#2021/10/18
#https://github.com/yotrew/commercial_skill_competition
n=int(input())

for j in range(n):
    a=input()
    b=input()
    dist=0
    for i in range(len(a)):
        if a[i]!=b[i]:
            dist+=1
    print(dist)

File: 110術科模擬試題/F.java
//#https://github.com/yotrew/commercial_skill_competition
import java.util.*;

public class F {
	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);
		String in1,in2;// = scanner.nextLine();
		int n =Integer.parseInt( scanner.nextLine());
				
		for(int i=0;i<n;i++){
			int dist=0;
			in1 = scanner.nextLine();
			in2 = scanner.nextLine();
			String[] ch1=in1.split("");
			String[] ch2=in2.split("");
			for(int j=0;j<ch1.length;j++){
				if(ch1[j].compareTo(ch2[j])!=0)
					dist+=1;
			}
			System.out.println(dist);
		}
		
	}
}
